clang/test/CIR/CodeGen/static-vars.cpp

```diff
@@ -35,3 +39,16 @@ void func2(void) {
   static float j;
   // CHECK-DAG: cir.global "private" internal dsolocal @_ZZ5func2vE1j = #cir.fp<0.000000e+00> : !cir.float
 }
+
+class b {
+public:
+  // CIR-AARCH64-DAG: cir.global linkonce_odr comdat @_ZZN1b4testEvE1c = #cir.int<0> : !s32i
+
+  // LLVM-AARCH64-DAG: $_ZZN1b4testEvE1c = comdat any
+  // LLVM-AARCH64-DAG: @_ZZN1b4testEvE1c = linkonce_odr global i32 0, comdat, align 4
+  void test() { static int c; }
+  // CIR-AARCH64-LABEL: @_ZN1b4testEv
+  // CIR-AARCH64: {{%.*}} = cir.get_global @_ZZN1b4testEvE1c : !cir.ptr<!s32i>
+};
```

Comment:
Just an static inside an inline function should do the trick: https://godbolt.org/z/8anK6G9x5